### ISSUE TITLE
Use the NextJS ESLint plugin to check for additional hidden issues in the code

### ISSUE THREAD

**Issue by bbenoist** [labelflow/labelflow#661]
https://nextjs.org/docs/basic-features/eslint

This is a follow-up of @RaynalHugo suggestion on another PR: https://github.com/labelflow/labelflow/pull/656#discussion_r770397940

> Every time we start the web app, it displays a warning saying we should probably use this.

We need to:

- [ ] Check whether the plugin can be useful
- [ ] See how we can use it in the context of a monorepo (calling `next lint` fells very invasive for example)
- [ ] Determine which warnings could be considered as errors

Depends on #656
